GoodFirstIssue-ToDoApp/Screens/ItemList/ItemListViewController.swift

```diff
@@ -10,11 +10,39 @@ import UIKit
 
 class ItemListViewController: UIViewController {
 
+    @IBOutlet private weak var tableView: UITableView! {
+        didSet {
+            tableView.register(ItemCell.loadNib(), forCellReuseIdentifier: ItemCell.reuseIdentifier)
+        }
+    }
+    
+    var itemList: [(String)] = []
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
     }
 
+}
+
+extension ItemListViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        itemList.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: ItemCell.reuseIdentifier, for: indexPath) as? ItemCell else { return UITableViewCell() }
```

Comment:
キャストできないときは、セルを新規で作って返すより、dequeueしたセルを返す方がいいのかなと思います（再利用するためのメソッドなので）。（今回はキャストできない場合はないので、一般論）

```suggestion
        let cell = tableView.dequeueReusableCell(withIdentifier: ItemCell.reuseIdentifier, for: indexPath)
        guard  let itemCell = cell as? ItemCell else {
            return cell
        }
```

Reply:
dequereReusableCell、決まり文句のような感じで書いていたのでそこまで考えてませんでした...
確かにその方がいいですね